docs/articles/expensify-classic/account-settings/Merge-Accounts.md

```diff
@@ -1,5 +1,33 @@
 ---
 title: Merge Accounts
-description: Merge Accounts
+description: How to merge two Expensify accounts and why this is useful.
 ---
-## Resource Coming Soon!
+
+# Overview
+
+Merging accounts allows you to combine two accounts. When you combine two accounts, all receipts, expenses, expense reports, invoices, bills, imported cards, secondary logins, co-pilots, and group policy settings will be combined into one account. 
+This can be useful if you start off with an account of your own but your organization creates a separate account for you. You can the track both personal and business expenses via one account. 
+
+# How to merge accounts
+Merging two accounts together is fairly straightforward. Let’s go over how to do that below:
+1. Navigate to expensify.com. 
+2. Log into the account you want to set as the Primary account.
+3. Navigate to Settings > Account > Account Details
+4. Scroll down to the Merge Accounts section and fill in the fields. Once you click Merge, a magic code link will be sent to you via email and you'll be prompted to enter the magic code
+5. Copy the magic code, switch back to the expensify.com page, and paste the code into the required field
+6. Click Merge Accounts
+If you have any questions about this process, feel free to reach out to Concierge for some assistance!
+
+# FAQ
+## Can you merge accounts from the mobile app?
+No, accounts can only be merged from the full website at expensify.com
+## Can I administratively merge two accounts together?
+No, only the account holder (user) can perform account merging.
+## Is merging accounts reversible?
+No, merging accounts is not reversible. It is a permanent action that cannot be undone.
+## Are there any restrictions on account merging?
+Yes! Please see below:
+If your email address belongs to a verified domain (verified in Expensify), you must start the process from the email account under the verified domain. You cannot merge a verified company email account into a personal account.
+If you have two accounts with two different verified domains, you cannot merge them together.
```

Comment:
```suggestion
* If your email address belongs to a verified domain (verified in Expensify), you must start the process from the email account under the verified domain. You cannot merge a verified company email account into a personal account.
* If you have two accounts with two different verified domains, you cannot merge them together.
```

I think this would look better as an unordered list!